tests/components/landisgyr_heat_meter/test_sensor.py

```diff
@@ -38,16 +38,16 @@ class MockHeatMeterResponse:
     """Mock for HeatMeterResponse."""
 
     heat_usage_gj: float
+    heat_usage_mwh: float
     volume_usage_m3: float
     heat_previous_year_gj: float
+    heat_previous_year_mwh: float
     device_number: str
     meter_date_time: datetime.datetime
 
 
 @patch(API_HEAT_METER_SERVICE)
-async def test_create_sensors(
-    mock_heat_meter, hass: HomeAssistant, entity_registry: er.EntityRegistry
-) -> None:
+async def test_sensors_gj(mock_heat_meter, hass: HomeAssistant) -> None:
```

Comment:
I think this change is not needed - maybe a bad rebase?

Reply:
Yes, thanks. Fixed this.